tests/libnet/ipaddress.go

```diff
@@ -0,0 +1,36 @@
+package libnet
+
+import (
+	k8sv1 "k8s.io/api/core/v1"
+	netutils "k8s.io/utils/net"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+func GetPodIpByFamily(pod *k8sv1.Pod, family k8sv1.IPFamily) string {
```

Comment:
This method is not in use by the PR so we may remove it.

Reply:
Done